CHANGELOG.unreleased.md

```diff
@@ -9,6 +9,7 @@
 
 - [REST API] Expose networks, VBDs, VDIs and VIFs
 - [Console] Supports host and VM consoles behind HTTP proxies [#6133](https://github.com/vatesfr/xen-orchestra/pull/6133)
+- [VM creation] Add link to documentation on secure boot [#6146](https://github.com/vatesfr/xen-orchestra/pull/6146)
```

Comment:
Don't forget to add `xo-web` to the "Packages to release".

Reply:
Rebase on `master` and it will be done for you.